src/components/SidebarMenu.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { MutableRefObject } from "react";
+import styles from "../styles/sidebarmenu.module.css";
+
+interface SidebarProps {
+  refs: MutableRefObject<HTMLElement | undefined>[];
+}
+
+export const SidebarMenu = ({ refs }: SidebarProps) => {
+  return (
+    <div className={styles.wrapper}>
+      <h4>Your new gang</h4>
+      <ul className={styles.ul}>
+        <a
```

Comment:
**nitpick:** We should render these links using `map` to have greater control of code, and prevent copy & pasting